src/utils/arch/riscv64/cpu.h

```diff
@@ -0,0 +1,33 @@
+/**
+* Copyright (c) 2001-2013, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
update copyright here as well.

Reply:
This file was already updated?

Reply:
No, I'm assuming you copied this line from other cpu.h file? Just need to update to 2023

Reply:
Understood & updated!